app/pages/learn/01_tutorial/07_rich_client_apps/02_introduction-javafx-animation.md

```diff
@@ -0,0 +1,265 @@
+- The `rate` defines the direction and speed at which the `Animation` is expected to be played. It supports both positive and negative numbers.
+- The `cycleCount` defines the number of cycles of this `Animation`. It can't be changed while running and must be positive.
+- The `cycleDuration` is the `Duration` of one cycle of this `Animation`. It is the time it takes to play from start to end of the `Animation` **at the default rate of 1.0**.
+- The `totalDuration` indicates the total duration of this `Animation`, including repeats. It is the result of `cycleDuration * cycleCount` or possibly `Duration.INDEFINITE`.
+- The `delay` is the `Duration` that delays the `Animation` when starting.
+- The `autoReverse` property specifies whether the `Animation` will play in reverse direction on alternating cycles.
+- The `onFinished` event handler is used to define additional behavior when the `Animation` finished.
+- The `status` represents the current state of the `Animation`, possible states are `PAUSED`, `RUNNING` and `STOPPED`.
+
+Additionally, it provides several useful methods, like `play()`, `playFrom(String cuePoint)`, `pause()`, `stop()` and more to control the animations flow.
+A quick look into the JavaDocs provides a great overview of its functionalities.
+
+## Transition
+The [Transition](javafxdoc:Transition) abstract class serves as the foundational class for all transitions, presenting a common form of `Animation`.
+JavaFX provides a variety of built-in transitions for common [Node](javafxdoc:Node) and [Shape](javafxdoc:Shape) properties.
+
+### Fade Transition
+The [FadeTransition](javafxdoc:FadeTransition) creates a fade effect. 
+This is done by updating the `opacity` property of the `Node` at regular intervals.
+
+![FadeTransition](/assets/images/javafx/animation/transition/fade-example.gif)
+```java
+Circle circle = new Circle(150, 150, 20, Color.GREEN);
+
+FadeTransition transition = new FadeTransition(Duration.seconds(5), circle);
+transition.setFromValue(1.0);
+transition.setToValue(0);
+transition.setInterpolator(Interpolator.LINEAR);
+
+transition.play();
+```
+
+### Fill Transition
+The [FillTransition](javafxdoc:FillTransition) creates an animation, that changes the filling of a shape. 
+This is done by updating the `fill` property of the `Shape` at regular intervals.
+
+![FillTransition](/assets/images/javafx/animation/transition/fill-example.gif)
+```java
+Circle circle = new Circle(150, 150, 20, Color.GREEN);
+
+FillTransition transition = new FillTransition(Duration.seconds(5), circle);
+transition.setFromValue(Color.GREEN);
+transition.setToValue(Color.BLACK);
+transition.setInterpolator(Interpolator.LINEAR);
+
+transition.play();
+```
+
+### Translate Transition
+The [TranslateTransition](javafxdoc:TranslateTransition) creates a move/translate animation from one position to another in a straight line.
+This is done by updating the `translateX`, `translateY` and `translateZ` properties of the `Node` at regular interval.
```

Comment:
```suggestion
This is done by updating the `translateX`, `translateY` and `translateZ` properties of the `Node` at regular intervals.
```

I know the docs say "at regular interval" but that's surely a copy-paste error. Let's make sure this article is correct. :) Can you fix that here and below?

Reply:
I posted this in our Slack and already got a reply:

> Yes, this is wrong. I'll file a bug.

I.e. it should be "at regular intervals".